tests/build_image_test.py

```diff
@@ -127,29 +120,7 @@ def test_channel_update_in_conda_env(mocker):
         side_effect=(lambda x: helpers.validate_cli(x, expect=["docker build",
                                                                "-t " + intended_image_name])))
 
-    channel_index_before, _ = get_channel_being_modified("tests/test-conda-env.yaml")
```

Comment:
I thought it was good to have this test to ensure conda env file is modified before build image. Any reason for removing this?

Reply:
We are removing the edited temporary environment file now and this test only worked assuming it remained there after the fact.

Reply:
You could mock the os.remove function to not do anything. That way we could still verify that the modifications are working properly.

Reply:
Good idea. I added that back with the mock.